sklearn/utils/tests/test_extmath.py

```diff
@@ -99,38 +99,50 @@ def test_logsumexp():
     assert_array_almost_equal(np.exp(logsumexp(logX, axis=1)), X.sum(axis=1))
 
 
-def test_randomized_svd_low_rank():
+def check_randomized_svd_low_rank(dtype):
```

Comment:
It's a convention in the projet to prefix by `check_` helper functions that are called by `test_` generators.